content/blog/posts/using-markdown-and-github-for-blogging-part-1.md

```diff
@@ -0,0 +1,72 @@
+
+4. **Non-authors can't edit the manuscript without authors' permission.** Only authors can make changes to the manuscript. This doesn't mean that they have all the power though because reviewers' approval is needed for publication (i.e., pull request merged).
+
+5. **Group communication is available.** Normally reviewers don't talk to each other but comments can make a group communication easier. This way, we can have a discussion and develop better ideas together. It also prevents reviewers from being siloed.
+
+6. **Review process is transparent**. Because of GitHub's great traceability, public repositories like ours will allow others to see our review process including the communication history.
+
+7. **Reviewing technical material is easy.** It's easy to incorporate code snippets, hyperlinks, etc. and they are all visible in the manuscript.
+
+8. **Everything exists in one place.** Normally non-document type files exist in a different place. However, this way, we can keep our blog post, documentation, and our code all in the same place.
+
+## How it works
+
+### Authors
+
+Write a blog post in a markup language by using any text editors. Once done, submit the manuscript for review as [a pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). At the pull request page, assign reviewers. To write a post in a markup language, you have to know the syntax. To submit it as pull request, you have to have a bit of knowledge on how to use git. 
+
+### Reviewers
+
+In a pull request page, you can check who are assigned as reviewers. If you are assigned, you will see your ID. [GitHub Docs](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) has detailed information on how to review changes. Keep in mind that even though you will have to read the manuscript in a markup language format, you can still check its rendered version by clicking "View File" option in the menu.
+
+### From pull request to merge
+
+Once the usual back-and-forth review process begins, reviewers make comments and requests for changes, and authors either accept them or make rebuttal. This process will be documented through comments and every piece of communication is tracked. Once everyone is satisfied, we make a collective decision to merge the pull request and it becomes a part of the master branch. Once this is done, we have several options. We can use several existing services to publish our repository directly as a website, or do something more sophisticated.
+
+## Coming soon
+
+In Part 2, our volunteer software engineer, [Nick Holden](https://github.com/nholden), will explain how we decided to use our existing content management system (CMS) to host the blog and publish posts. This approach provides consistency in development, and convenience in publication.
```

Comment:
Breaking this into two separate blog posts makes a lot of sense to me, and I'm happy to write about the topic you describe here! I'm not sure it's necessary to preview it here though. What do you think about leaving this bit off? We can always add a link to part 2 here once it's published.

Reply:
Sure that works! I will leave this part off. Thanks for the feedback, Nick :)